code/modules/projectiles/guncrafting/attachments/energy_bullets.dm

```diff
@@ -0,0 +1,46 @@
+/obj/item/weapon/gun_attachment/energy_bullet
+	gun_type = CUSTOMIZABLE_ENERGY
+	uses_overlay = FALSE
+	icon_state = "bullet"
+	not_okay = /obj/item/weapon/gun_attachment/energy_bullet
+
+/obj/item/weapon/gun_attachment/energy_bullet/disorienter
+	name = "Energy Disorienter"
+	desc = "Drowsiness, less damage."
+
+/obj/item/weapon/gun_attachment/energy_bullet/disorienter/on_fire(var/obj/item/weapon/gun/owner, var/obj/item/projectile/bullet)
+	..()
+	bullet.drowsy += 10
```

Comment:
Mmm, at the moment the disorienter can actually make people fall asleep extremely quickly, and that also makes its name a bit misleading. I think you should either set the drowsy to 4 or 5, or should should make that it instead makes the victim dizzy.